docs/tutorials/qiskit_pulse.rst

```diff
@@ -46,7 +46,7 @@ First, we use the pulse module in Qiskit to create a pulse schedule.
     w = 5.
 
     # Sample rate of the backend in ns.
-    dt = 0.222
+    dt = 1/4.5
```

Comment:
```suggestion
    dt = 1 / 4.5
```

`black` and `lint` don't detect things in the tutorials, so we need to manually catch style issues here. Binary operators should be surrounded by spaces.

Reply:
Thanks. I changed at https://github.com/Qiskit/qiskit-dynamics/pull/188/commits/591d80d8d8174110ad2d46cce47c0bb78a3f0a06